### PR TITLE
Feature navigation

### PR DESCRIPTION
**Pull request by irenepapaspyratos**

### CODE REVIEW

File: src/services/get-staticQueries.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+export default function getStaticQueries(string, queryElement, queryArrayA, queryArrayB) {
```
Comment (aclaes) [inline src/services/get-staticQueries.js:1]: Naming a variable just `string` is very unintuitive. Please consider using a more describing name, such as `key` or `subject`

**Review by aclaes — CHANGES_REQUESTED** [review]

File: pages/list/[key]/[name].jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		return urlAdjustLight(element.name) === router.query.name && element.id;
+	});
+
+	useEffect(() => {
+		fetchData(keyArray);
+	}, [fetchData, keyArray]);
+
+	return (
+		<>
+			<RenderIf isTrue={key === 'spot'}>
```
[the remark this answers is not in the document]
Comment (irenepapaspyratos) [reply pages/list/[key]/[name].jsx:28]: 😔 Okeeeeey.....

File: pages/list/[key]/[name].jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+				<Spot
+					index={indexFound}
+					spots={data.spots}
+					animals={data.animals}
+					organizers={data.organizers}
+				/>
+			</RenderIf>
+
+			<RenderIf isTrue={key === 'animal'}>
```
[the remark this answers is not in the document]
Comment (irenepapaspyratos) [reply pages/list/[key]/[name].jsx:37]: 😔 Okeeeeey....

File: src/components/ElementList/ElementList.jsx [hunk 4]
```diff
@@ -1,14 +1,32 @@
...
+				<summary>{displayHeader}</summary>
+				<ul>
+					{elements.map(element => {
+						return (
+							<li key={element.id}>
+								<Link
+									href={{
+										pathname: `/list/${urlPart}/[slug]`,
+										query: { slug: urlAdjustLight(element.name) },
+									}}
```
[the remark this answers is not in the document]
Comment (irenepapaspyratos) [reply src/components/ElementList/ElementList.jsx:21]: Cool! Thank you!!

File: src/services/url-adjust-light.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export default function urlAdjustLight(string) {
+	return string.toLowerCase().replace(' ', '-');
```
[the remark this answers is not in the document]
Comment (irenepapaspyratos) [reply src/services/url-adjust-light.js:2]: Mhhh... when I do this, the shown url isn't nice to read anymore for the user. Under the coat the url is encoded anyway by next - therefore I decided to use this method.
But if you tell me to change it with encoding, I will follow.

[the remark this answers is not in the document]
Comment (aclaes) [reply src/services/url-adjust-light.js]: You might run into trouble if lets say an organizer’s name includes certain special chars (like `&`). You can keep it and adjust if needed.

**Review by aclaes — APPROVED** [review]

**Pull request merged**